sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusSessionManagerTest.java

```diff
@@ -155,7 +155,7 @@ void afterEach(TestInfo testInfo) {
     @Test
```

Comment:
Where are the unit tests for Session receiver?

Reply:
Added unit test for ServiceBusSessionReceiverAsyncClient and ServiceBusSessionReceiverClient.